tests/logs/RegressionTests_hera.log

```diff
@@ -7556,25 +7556,30 @@ Checking test 230 cpld_debug_pdlib_p8_gnu results ....
  Comparing 20210322.090000.out_pnt.ww3 .........OK
  Comparing 20210322.090000.out_grd.ww3 .........OK
 
-  0: The total amount of wall time                        = 919.218809
-  0: The maximum resident set size (KB)                   = 1296864
+  0: The total amount of wall time                        = 903.828440
+  0: The maximum resident set size (KB)                   = 1303532
 
 Test 230 cpld_debug_pdlib_p8_gnu PASS
 
 
 baseline dir = /scratch1/NCEPDEV/nems/emc.nemspara/RT/NEMSfv3gfs/develop-20230525/datm_cdeps_control_cfsr_gnu
-working dir  = /scratch1/NCEPDEV/stmp2/Zachary.Shrader/FV3_RT/rt_254496/datm_cdeps_control_cfsr_gnu
+working dir  = /scratch1/NCEPDEV/stmp2/Anning.Cheng/FV3_RT/rt_25084/datm_cdeps_control_cfsr_gnu
 Checking test 231 datm_cdeps_control_cfsr_gnu results ....
  Comparing RESTART/20111002.000000.MOM.res.nc .........OK
  Comparing RESTART/iced.2011-10-02-00000.nc .........OK
  Comparing RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc .........OK
 
- 0: The total amount of wall time                        = 174.963796
- 0: The maximum resident set size (KB)                   = 665672
+ 0: The total amount of wall time                        = 179.618159
+ 0: The maximum resident set size (KB)                   = 661592
 
 Test 231 datm_cdeps_control_cfsr_gnu PASS
 
+FAILED TESTS: 
+051 merra2_thompson_intel failed in check_result
+merra2_thompson_intel 051 failed in run_test
+172 atmaero_control_p8_rad_micro_intel failed in check_result
+atmaero_control_p8_rad_micro_intel 172 failed in run_test
```

Comment:
@AnningCheng-NOAA Is the failure to the additional test expected? You listed only the merra2_thompson test in the expected test changes.

Reply:
@AnningCheng-NOAA can you resolve this conversation? @grantfirl can you make comment about this conversation?